services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -170,6 +182,117 @@ async def upload_files(files: list[UploadFile] = FileParam(...)):
     raise NotImplementedError
 
 
+@router.post(
+    "/content",
+    response_model=FileUploadSchema,
+)
+@cancel_on_disconnect
+@catch_n_raise(
+    (NodeportsException, ValidationError, ClientError),
+    lambda e: status.HTTP_500_INTERNAL_SERVER_ERROR,
+)
+async def get_upload_links(
+    request: Request,
+    client_file: ClientFile,
+    user_id: Annotated[PositiveInt, Depends(get_current_user_id)],
+):
+    """Get upload links for uploading a file to storage
+
+    Arguments:
+        request -- Request object
+        client_file -- ClientFile object
+        user_id -- User Id
+
+    Returns:
+        FileUploadSchema
+    """
+    assert request  # nosec
+    file_meta: File = await File.create_from_client_file(
+        client_file,
+        datetime.datetime.now(datetime.timezone.utc).isoformat(),
+    )
+    _, upload_links = await get_upload_links_from_s3(
+        user_id=user_id,
+        store_name=None,
+        store_id=SIMCORE_LOCATION,
+        s3_object=parse_obj_as(
+            StorageFileID, f"api/{file_meta.id}/{file_meta.filename}"
+        ),
+        client_session=None,
+        link_type=LinkType.PRESIGNED,
+        file_size=ByteSize(client_file.filesize),
+        is_directory=False,
+    )
+    return upload_links
+
+
+@router.patch(
```

Comment:
sorry I forgot to share this with you. these are the guidelines we try to follow: https://cloud.google.com/apis/design/custom_methods#http_mapping

in case you would use PATCH call you would need a resource such as:
PATCH /content/{content_id}

that would be correct and more understandable I think.

other option would be:
POST /content/{content_id}:complete
that is what the storage API actually allows.

Reply:
Fixed. Here are the new endpoints:
![image](https://github.com/ITISFoundation/osparc-simcore/assets/126242332/91fa753d-70b1-4179-835d-2e61f4eceafe)